packages/pass-style/package.json

```diff
@@ -37,6 +37,7 @@
     "@fast-check/ava": "^1.1.5"
   },
   "devDependencies": {
+    "@endo/eventual-send": "^0.17.6",
```

Comment:
Yeah.  `@endo/pass-style` really does seem like it should be home for `getMethodNames`.